cliv2/internal/cliv2/cliv2.go

```diff
@@ -218,7 +217,7 @@ func (c *CLI) GetBinaryLocation() string {
 }
 
 func (c *CLI) GetTempDir() string {
-	return local_utils.GetTemporaryDirectory(c.CacheDirectory, cliv1.CLIV1Version())
+	return utils.GetTemporaryDirectory(c.CacheDirectory, cliv1.CLIV1Version())
```

Comment:
Issue: with the new gaf version, the temp directory is a configuration value. Instead of using this helper function directly, we should be using the configuration now, otherwise the behaviour wouldn't be consistent.

```suggestion
	return c.globalConfig.GetString(configuration.TEMP_DIR_PATH)
```